src/components/Loader.vue

```diff
@@ -139,14 +145,22 @@ class Loader extends Vue {
      */
     private showLoadingBackground: boolean = true;
 
-    private _loadingTitle?: string;
+    private loadingTitle?: string = '';
```

Comment:
```suggestion
    private loadingTitle: string = '';
```
is sufficient I think.

Reply:
:+1: